src/appkit.cs

```diff
@@ -21865,12 +21865,12 @@ partial interface NSOutlineView : NSAccessibilityOutline {
 		void RemoveItems (NSIndexSet indexes, [NullAllowed] NSObject parent, NSTableViewAnimationOptions animationOptions);
 
 		// - (void)insertRowsAtIndexes:(NSIndexSet *)indexes withAnimation:(NSTableViewAnimationOptions)animationOptions UNAVAILABLE_ATTRIBUTE;
-		[Availability (Unavailable = Platform.iOS_Version | Platform.Mac_Version)]
+		[Availability (Unavailable = Platform.Mac_Version)]
```

Comment:
This is giving me an apidiff change locally on the main PR but I don't understand what's up.

Reply:
What diff is it showing?